worker/src/main/scala/org/apache/celeborn/service/deploy/worker/Worker.scala

```diff
@@ -723,13 +728,14 @@ private[celeborn] class Worker(
           removeAppActiveConnection(applicationId)
           secretRegistry.unregister(applicationId)
         }
-        logInfo(s"Cleaned up expired shuffle $shuffleKey")
       }
+      logInfo(s"Cleaned up expired shuffle keys $cleanShuffleKeys")
       partitionsSorter.cleanup(expiredShuffleKeys)
       fetchHandler.cleanupExpiredShuffleKey(expiredShuffleKeys)
       threadPool.execute(new Runnable {
         override def run(): Unit = storageManager.cleanupExpiredShuffleKey(expiredShuffleKeys)
```

Comment:
The most time-consuming part of cleaning expired shuffles is here. Recording this time would be meaningful.